content/blog/2016/2016-06-11-gsoc-jenkins-support-core-plugin-improvements.adoc

```diff
@@ -0,0 +1,82 @@
+
+https://www.linkedin.com/in/minudika[LinkedIn] | https://twitter.com/minudika[Twitter]
+
+=== Abstract
+
+The Support-Core Plugin provides the basic infrastructure for generating "bundles" of support information with Jenkins.
+There are two kinds of bundles.
+
+* Automatic bundles : bundles which are get saved in $JENKINS_HOME/support once per hour starting 15 seconds after Jenkins starts the plugin.
+The automatic bundles are retained using an exponential aging strategy. Therefore it's possible to have a bunch of them over the entire lifetime after the plugin installing the plugin.
+
+* On demand bundles : these bundles are generated from the root "Support" action.
+
+However current support-core plugin is not much user friendly. The object of this project is to make it more user friendly by adding some features which make a sophisticated environment for the user who use support plugin.
+
+In this project scope, there are three features and improvements we are going to consider.
+
+. Ease the management bundles by the administrator (https://issues.jenkins-ci.org/browse/JENKINS-33090[JENKINS-33090])
+. Adding an option to anonymize customer labels (strings created by the user such as name of  a job, folder, view, slave, and template etc). (https://issues.jenkins-ci.org/browse/JENKINS-33091[JENKINS-33091])
+. Allowing user to create an issue and submit a bundle into the OSS tracker using the support-core plugin. (https://issues.jenkins-ci.org/browse/JENKINS-21670[JENKINS-21670])
+
+https://github.com/aheritier[Arnaud Heritier] and https://github.com/christ66[Steven Christou] are guiding me through the project as my mentors.
+
+image:/images/post-images/gsoc-support-core-plugin/generate-bundle.png[Popup window, role=center]
+
+
+=== Tasks and Deliverables
+
+
+==== Ease the management bundles by the administrator.
+
+Under this task, the following functionalities are going to be implemented.
+
+. Listing bundles stored on the jenkins instance with their details.
+. Allowing user to download each bundle.
+. Allowing user to delete each bundles or all bundles.
+. Allowing user to browse the content of each bundles.
+. Automatically purging old bundles.
+
+image:/images/post-images/gsoc-support-core-plugin/downloadBundles.png[Popup window, role=center]
+image:/images/post-images/gsoc-support-core-plugin/config.png[Popup window, role=center]
+
+==== Enabling user to create an issue and submit a bundle into the OSS tracker
+
+When a Jenkins user sees an issue , he/she commonly contacts his support contacts (Jenkins instance admins)  and then Jenkins admins troubleshoot the issue.
+The objective of this task is to implement a feature which enables the user to report an issue to a admin through support core plugin.
+
+When we create bundles to attach with the ticket, it is important to protect user’s privacy who creates the ticket. When considering doing that, anonymizing user created labels (texts) comes to the front.
```

Comment:
I'm no native speaker, but I think _protect user’s privacy who creates the ticket_ is missing a word. Something like "protect user’s privacy of *the one* who creates the ticket"

?